script/Ciel/Day 15/15_01B_CIEL15B_1.txt

```diff
@@ -2606,7 +2611,7 @@ I open my eyes.
 [sha:a0ee6641a73002b871a5d29e86fd0b4091440216]{
 -- Page 154, Offset 42433.
 -- 目の前には、彼女の、泣き顔があった。
-When I do, there before me is her crying face.
+And there before them, she is, her face in tears.
```

Comment:
Just revert.

Reply:
prev boring

Reply:
Sorry but this structure just doesn't work in EN. The original is fine.

Reply:
??